Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Handler.yml

```diff
@@ -241,20 +241,216 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "7":
+    continueonerrortype: ""
+    form:
+      description: Add one or more tags to the assets(s) associated with the current Cortex XSOAR incident.
+      expired: false
+      questions:
+      - defaultrows: []
+        fieldassociated: ""
+        gridcolumns: []
+        id: "0"
+        label: ""
+        labelarg:
+          simple: What tag(s) do you want to add to asset(s)?
+        options: []
+        optionsarg: []
+        placeholder: ""
+        readonly: false
+        required: false
```

Comment:
Should we require the user to add tags and not leave it empty?
![image](https://github.com/demisto/content/assets/88268646/67efed8c-b8ea-4db3-ac87-f059053d9abd)

Reply:
changed in
[sasha recommendations](https://github.com/demisto/content/pull/35111/commits/1d4310c1fe899d397e962890e60dd244edd74b83)